setup.cfg

```diff
@@ -3,6 +3,7 @@ python_files = evalml/tests/*
 filterwarnings =
     ignore::DeprecationWarning
     ignore::PendingDeprecationWarning
+junit_duration_report = call
```

Comment:
This flag is set so that we don't report setup/teardown time, just the call time: https://docs.pytest.org/en/6.2.x/reference.html#confval-junit_duration_report